### ISSUE TITLE
BUG: cls-hooked not working

### ISSUE THREAD

**Issue by muxiangqiu** [stackblitz/webcontainer-core#196]
**Describe the bug:**
cls-hooked not working.

**Link to the blitz that caused the error:**
https://stackblitz.com/edit/node-zwyb6w?file=index.js

**Steps to reproduce:**

1. Go to https://stackblitz.com/edit/node-zwyb6w?file=index.js
2. node index.js
3. See error

**Expected behavior:**
I would expect the command to succeed without error(s)

**Screenshots:**
![image](https://user-images.githubusercontent.com/11471925/122159489-02fb1d00-cea1-11eb-9806-1bb4323b5aee.png)

**Desktop (please complete the following information):**
 - Device: Intel MacBook Pro
 - OS: OS X 10_15_7
 - Browser: Chrome
 - Version 91.0.4472.101

**Comment by muxiangqiu** [comment]
Can someone help me solve it？

**Comment by SamVerschueren** [comment]
Hi @muxiangqiu! I don't think we currently have anything implemented regarding `async_hooks`. Chiming in @d3lm as he might have a better view on this.

**Comment by d3lm** [comment]
Yes that's not implemented yet.

**Comment by muxiangqiu** [comment]
Got it! Thanks!

It would be great if webcontainer can support `async_hooks` .

**Comment by d3lm** [comment]
Yep for sure. We'll add this to our roadmap but we cannot promise any date when this will be ready. We'll keep you posted if there's updated on this. Thanks for your understanding 🙏

**Comment by HeyGarrison** [comment]
@d3lm It looks like we ended up getting in this enhancement? The project seems to run? 🤷

**Comment by d3lm** [comment]
Not sure if it's unrelated now to `async_hooks` but that is still unsupported for the time being.

**Comment by muxiangqiu** [comment]
<img width="1533" alt="image" src="https://user-images.githubusercontent.com/11471925/154026917-5b1eb845-2abe-438e-aa6c-b0ca47bc119d.png">
